public/docs/ts/latest/guide/glossary.jade

```diff
@@ -0,0 +1,445 @@
+    <label>My current hero is {{hero.name}}</label>
+
+  :markdown
+    Learn more about interpolation in the 
+    [Template Syntax](./template-syntax.html#interpolation) chapter.
+    
+.l-main-section
+:markdown
+  ## Output
+.l-sub-section
+  :markdown
+    A directive property that can be the ***target*** of an 
+    [Event Binding](./template-syntax.html#property-binding).
+    Events stream *out* of this property to the receiver identified
+    in the template expression to the right of the equal sign.
+    
+    See the [Template Syntax](./template-syntax.html#inputs-outputs) chapter.
+    
+.l-main-section
+:markdown
+  ## Pipe
+.l-sub-section
+  :markdown
+    An Angular pipe is a function that transforms input values to output values for
+    display in a [view](#view). We use the `@Pipe` [decorator](decorator)
+    to associate the pipe function with a name. We then can use that
+    name in our HTML to declaratively transform values on screen.
+    
+    Here's an example that uses the built-in `currency` pipe to display
+    a numeric value in the local currency. 
+    
+  code-example(language="html" escape="html").
+    <label>Price: </label>{{product.price | currency}}
+  :markdown
+    Learn more in the chapter on [pipes](./pipes.html) .
+    
+:markdown
+  ## Provider
+.l-sub-section
+  :markdown
+    Angular relies on [dependency injection](#dependency-injection) to create and deliver
+    pieces of functionality to the application parts that need them. Those pieces are
+    often called "dependencies". 
+    
+    Angular can't find or create these dependencies by itself. Something has to "provide" them.
+    That something is called a "Provider". A provider can be almost anything that can produce
+    the dependency although it is most often a class that we register with the 
+    [dependency injector](injector). 
+    
+    We typically register our application providers when we [bootstrap](#bootstrap) the application
+    but there are other opportunities to do that as well. 
+    See the [Dependency Injection[(./dependency-injection.html)] chapter for details.
+ 
+.l-main-section
+:markdown
+  ## Router
+.l-sub-section
+  :markdown
+    Most applications consist of many screens or [views](#view).
+    The user navigates among them by clicking links and buttons
+    and taking other similar actions that cause the application to
+    replace one view with another.
+    
+    The [Angular router](./router.html) is a richly featured mechanism for configuring
+    and managing the entire navigation process including the creation and destruction
+    of views.
+        
+.l-main-section
+:markdown
+  ## Structural Directive
+.l-sub-section
+  :markdown
+    A category of [Directive](#directive) that can 
+    shape or re-shape HTML layout, typically by adding, removing, or manipulating
+    elements and their children.
+    
+    The `ng-if` "conditional element" directive and the `ng-for` "repeater" directive are
```

Comment:
They are directives without the * and may even be applied without the *, albeit in the expanded form with the `<template>` tag.

I have played fast and loose with directive names, sometimes using the class name (`NgIf`) and sometimes the applied form (`ng-if`). Someone may want to clean this up everywhere when if/when we detect confusion in the ranks.